app/components/navigation_header/index.tsx

```diff
@@ -62,13 +62,12 @@ const NavigationHeader = forwardRef<SearchRef, Props>(({
 
     const {largeHeight, defaultHeight, headerOffset} = useHeaderHeight();
     const containerHeight = useAnimatedStyle(() => {
-        const minHeight = defaultHeight;
         const value = -(scrollValue?.value || 0);
         const calculatedHeight = (isLargeTitle ? largeHeight : defaultHeight) + value;
         const height = lockValue?.value ? lockValue.value : calculatedHeight;
         return {
-            height: Math.max(height, minHeight),
-            minHeight,
+            height: Math.max(height, defaultHeight),
+            minHeight: defaultHeight,
```

Comment:
I don't see how this change makes things any different. Can you explain?

Reply:
Hi @larkox, this change was not related to the issue, it was only to simplify code.